packages/fast-usdc-deploy/src/fast-usdc-update.build.js

```diff
@@ -51,14 +54,25 @@ export const updateProposalBuilder = async (
 export default async (homeP, endowments) => {
   const { writeCoreEval } = await makeHelpers(homeP, endowments);
   const {
-    values: { feedPolicy },
+    values: { feedPolicy: feedPolicyJSON, network },
   } = parseArgs({ args: endowments.scriptArgs, options });
 
-  const parseFeedPolicy = () => {
-    if (typeof feedPolicy !== 'string') throw Error(feedPolicyUsage);
-    return JSON.parse(feedPolicy);
-  };
-  const config = harden({ feedPolicy: parseFeedPolicy() });
+  if (typeof feedPolicyJSON !== 'string') throw Error(feedPolicyUsage);
+  const feedPolicy = JSON.parse(feedPolicyJSON);
+  if (network !== undefined) {
+    if (typeof network !== 'string') {
+      throw Error('--network requires a string argument');
+    }
+    if (!Object.hasOwn(ChainPolicies, network)) {
```

Comment:
Given that `ChainPolicies` is a design-time constant, I suggest the more straightforward:

```suggestion
    if (!(network in ChainPolicies)) {
```

Reply:
or to get diagnostics for free:

```suggestion
    mustMatch(network, M.or(...Object.keys(ChainPolicies)))
```